test/bin/force_vm_settings.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -x
+#
+# This script is used by scenario.sh when creating a new VM to force
+# specific settings on the host, such as opening firewall ports.
+
+set -euo pipefail
+
```

Comment:
If there's an assumption that we run with SUID==0, should we check it in the script?

Reply:
This script is invoked from `scenario.sh` using sudo. It's not meant to be used directly.

Reply:
We might want to either add sudo in the script or check for SUID==0 for consistency.
I'm sure we'd want to run this script directly because it's conveniently setting up the environment.

Reply:
Done